.github/workflows/stable-publish.yml

```diff
@@ -1,9 +1,8 @@
 name: Publish stable
 
 on:
-  push:
-    branches:
-      - master
+  workflow_dispatch: {}
+  workflow_call: {}
```

Comment:
keep on push branches here and remove workflow_dispatch, it is only for stable-push , this should detect the author and if the commit message has rc or not